sdk/freertos_app_cpu1/.project

```diff
@@ -1,38 +1,43 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<projectDescription>
-	<name>freertos_app_cpu1</name>
-	<comment>Created by SDK v2019.1. amdc_freertos_bsp_cpu1 - ps7_cortexa9_1</comment>
-	<projects>
-		<project>amdc_freertos_bsp_cpu1</project>
-	</projects>
-	<buildSpec>
-		<buildCommand>
-			<name>org.eclipse.cdt.managedbuilder.core.genmakebuilder</name>
-			<arguments>
-			</arguments>
-		</buildCommand>
-		<buildCommand>
-			<name>org.eclipse.cdt.managedbuilder.core.ScannerConfigBuilder</name>
-			<triggers>full,incremental,</triggers>
-			<arguments>
-			</arguments>
-		</buildCommand>
-	</buildSpec>
-	<natures>
-		<nature>org.eclipse.cdt.core.cnature</nature>
-		<nature>org.eclipse.cdt.managedbuilder.core.managedBuildNature</nature>
-		<nature>org.eclipse.cdt.managedbuilder.core.ScannerConfigNature</nature>
-	</natures>
-	<linkedResources>
-		<link>
-			<name>FreeRTOS-Kernel</name>
-			<type>2</type>
-			<location>S:/School/WEMPEC/firmware-rtos-e/sdk/FreeRTOS-Kernel</location>
-		</link>
-		<link>
-			<name>shared</name>
-			<type>2</type>
-			<location>S:/School/WEMPEC/firmware-rtos-e/sdk/shared</location>
-		</link>
-	</linkedResources>
-</projectDescription>
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>freertos_app_cpu1</name>
+	<comment>Created by SDK v2019.1. amdc_freertos_bsp_cpu1 - ps7_cortexa9_1</comment>
+	<projects>
+		<project>amdc_bsp_cpu1</project>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.cdt.managedbuilder.core.genmakebuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.cdt.managedbuilder.core.ScannerConfigBuilder</name>
+			<triggers>full,incremental,</triggers>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.cdt.core.cnature</nature>
+		<nature>org.eclipse.cdt.managedbuilder.core.managedBuildNature</nature>
+		<nature>org.eclipse.cdt.managedbuilder.core.ScannerConfigNature</nature>
+	</natures>
+	<linkedResources>
+		<link>
+			<name>FreeRTOS-Kernel</name>
+			<type>2</type>
+			<location>C:/Users/srich008/Information/v2-staging/sdk/FreeRTOS-Kernel</location>
+		</link>
+		<link>
+			<name>MemMang</name>
+			<type>2</type>
+			<location>C:/Users/srich008/Information/v2-staging/sdk/FreeRTOS-Kernel/portable/MemMang</location>
+		</link>
+		<link>
+			<name>shared</name>
+			<type>2</type>
+			<location>C:/Users/srich008/Information/v2-staging/sdk/shared</location>
+		</link>
```

Comment:
We'll need to clean this up to not have absolute paths on your computer.

We can change these with relative project paths if you are careful in the SDK wizard...

For now, let's ignore this and keep moving forward.

Reply:
Yes, I added a similar comment on this file in the `cpu0` app.